Examples/Tests/collision/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ add_warpx_test(
     RZ  # dims
     1  # nprocs
     inputs_test_rz_collision  # inputs
-    analysis_collision_rz.py  # analysis
-    diags/diag1000150  # output
+    "analysis_collision_rz.py diags/diag1000150"  # analysis
+    "analysis_default_regression.py --path diags/diag1000150 --plotfile --skip-particle"  # checksum
```

Comment:
Typo? `--skip-particles`

Reply:
Thanks, fixed in [b57c6f1](https://github.com/ECP-WarpX/WarpX/pull/5456/commits/b57c6f19c3c3bdb66f2d180ead16d20a7b277285).